modin/experimental/xgboost/xgboost.py

```diff
@@ -52,25 +54,35 @@ def __init__(self, data, label):
         self.label = unwrap_partitions(label, axis=0)
 
     def __iter__(self):
+        """Return unwrapped `self.data` and `self.label`.
+
+        Yields
+        ------
+        list
+            List of `self.data` with pairs of references to IP of row partition
+            and row partition [(IP_ref0, partition_ref0), ..].
+        list
+            List of `self.label` with pairs of references to row partition
+            [partition_ref0, partition_ref1, ..].
```

Comment:
this example looks strange, maybe you wanted to write `[(partition_ref0, partition_ref1), ..]`?

Reply:
rewritten by the next:
```
List of `self.label` with references to row partitions [partition_ref0, ..].
```